packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -61,8 +83,9 @@ export class TableInputCSV extends React.PureComponent<Props, State> {
   render() {
     const { width, height } = this.props;
     const { data } = this.state;
+    const styles = getStyles();
```

Comment:
As a matter of consistency please use `useStyles` hook for retrieving the styles. You can find some more info about this in the [styling documentation](https://github.com/grafana/grafana/blob/master/contribute/style-guides/styling.md#styling-with-theme)

Reply:
Gotcha 😀🙌

Reply:
It seems like I can't use the `useStyles` hook inside a class component.
Is there a workaround to this? I'm considering converting the component from a class to a functional component. WDYT? @dprokop

Reply:
You're correct about hooks, Uche. But instead of converting this into a Functional component, (which is a splendid exercise for another time) you should use the `stylesFactory` function. There are plenty of examples of stylesFactory and Class components like [https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/Logs/LogLabelStats.tsx](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/Logs/LogLabelStats.tsx) for instance. Study how that Components interface `extends Themable` and uses `withTheme`.

I've also noticed that you have a lot of PRs opened at the same time now. I think you're wasting time context switching between these.
Try to focus on the task at hand i.e. migrate the styles and closing this PR. Then move on to the next PR, read all the comments, fix them, test the PR and try to get that PR closed and repeat.

Reply:
Alright